Chutzpah/Utility/ValidationHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chutzpah.Utility
+{
+    public class ValidationHelper
+    {
+        private const string ValidProxyPattern = @".*:[\d]+$";
+        public static bool IsValidProxySetting(string value)
```

Comment:
@sunnyy999 This method is called IsValidProxy but seems to return true if it is Invalid actually. And looking in other changes it seems sometimes the code is using it to mean valid and other times invalid.